modules/e2ee/Context.js

```diff
@@ -362,9 +440,7 @@ export class Context {
                 counter,
                 length: CTR_LENGTH
             }, this._cryptoKeyRing[keyIndex].encryptionKey, new Uint8Array(encodedFrame.data,
-                    UNENCRYPTED_BYTES[encodedFrame.type],
-                    encodedFrame.data.byteLength - (UNENCRYPTED_BYTES[encodedFrame.type]
-                    + DIGEST_LENGTH[encodedFrame.type] + counterLength + signatureLength + 1))
+                    UNENCRYPTED_BYTES[encodedFrame.type], authTagOffset - UNENCRYPTED_BYTES[encodedFrame.type])
```

Comment:
this is nicer than in the old version. It says
"start at offset, then decrypt until the authentication tag"